Extension/src/LanguageServer/client.ts

```diff
@@ -413,7 +413,8 @@ class DefaultClient implements Client {
         let settings: CppSettings = new CppSettings(workspaceFolder ? workspaceFolder.uri : null);
         let other: OtherSettings = new OtherSettings(workspaceFolder ? workspaceFolder.uri : null);
 
-        let storagePath: string = util.extensionContext.storagePath;
+        let storagePath: string = util.extensionContext ? util.extensionContext.storagePath :
+            path.join((workspaceFolder ? workspaceFolder.uri.fsPath : ""), "/.vscode");
```

Comment:
I'm not sure we want to change this.  The extension binary does something different when there is no storage path, right?